tools/go-agent/config/loader.go

```diff
@@ -282,6 +282,20 @@ func (s *StringValue) GetListStringResult() []string {
 	return strings.Split(val, ",")
 }
 
+// GetListStringResultByLookup get the value from os.LookupEnv, if the env found, even it's empty
+func (s *StringValue) GetListStringResultByLookup() []string {
```

Comment:
Shall we test this function? I know it is very simple but I can't find a good reason for not to test it.